test/msw-api/auto-update-worker.test.ts

```diff
@@ -26,12 +27,12 @@ afterAll(async () => {
 
 test('updates the worker script on the postinstall hook', async () => {
   // Pack the current state of the "msw" package.
-  execSync(`yarn pack --filename ${fsMock.resolve('msw.tgz')}`, {
+  execSync(`pnpm pack --pack-destination ${fsMock.resolve('.')}`, {
```

Comment:
PNPM doesn't support a filename destination but the directory. Need to derive the package name from the version in package.json.